recipes/util-is/src/workflow.ts

```diff
@@ -69,7 +69,7 @@ const replacements = new Map<string, (arg: string) => string>([
  * 5. util.isError() → value instanceof Error
  * 6. util.isFunction() → typeof value === 'function'
  * 7. util.isNull() → value === null
- * 8. util.isNullOrUndefined() → value == null
+ * 8. util.isNullOrUndefined() → value === null || value === undefined
```

Comment:
very nit since it's a comment:

```suggestion
 * 8. util.isNullOrUndefined() → value == null
```

Reply:
Yes, I understand :)

```js
     null == null // true
undefined == null // true
    false == null // false
        0 == null // false
       '' == null // false
```

Reply:
The difference is in `document.all`, I believe, which doesn't apply to node but would if someone cargoculted the info into a browser.

Reply:
`document.all == null` but `document.all !== null && typeof document.all !== 'undefined'`.